client-ts/Microsoft.AspNetCore.SignalR.Client.TS/Transports.ts

```diff
@@ -196,6 +207,8 @@ export class LongPollingTransport implements ITransport {
             }
             else {
                 if (this.onClosed) {
+                    // TODO: Uncaught DOMException: Failed to read the 'responseText' property from 'XMLHttpRequest':
+                    // The value is only accessible if the object's 'responseType' is '' or 'text' (was 'arraybuffer').
                     this.onClosed(new Error(`Status: ${pollXhr.status}, Message: ${pollXhr.responseText}`));
```

Comment:
@mikaelm12 - this tells me that putting statusText to error message is more tricky than it seems.